tests/unit/layers/models/layersManager.spec.ts

```diff
@@ -35,7 +35,7 @@ describe('layersManager', () => {
   describe('#getLayer', () => {
     it('should successfully return the requested layer', async () => {
       // action
-      const resource: IMapProxyCache = await layersManager.getLayer('mockLayerNameExists');
+      const resource: IMapProxyCache = await layersManager.getLayer('mockLayerNameExists-source');
```

Comment:
please add unit tests for the section that decides if to use redis layer and other methods if needed